src/core/node/node.ts

```diff
@@ -471,6 +471,7 @@ export class Node extends Config {
           recordIds.includes(record.id)
         ),
       ];
+      this.records.forEach((record: NodeRecord) => record.update());
```

Comment:
(please see `nodeRecord.ts`)
```suggestion
      this.records.forEach((record: NodeRecord) => record.updateGroupID());
```